### PR TITLE
Update profile page and allow user editing name, description, and username

### LINKED ISSUE #33: Edit profile

### PR DESCRIPTION
**Pull request by sbeleidy**
Fixes #33

### CODE REVIEW

**Comment by sbeleidy** [comment]
I think there's something wrong with the preview. So check out the code locally.

File: pages/[username].js [hunk 1]
```diff
@@ -21,6 +23,17 @@ export default function ProfilePage() {
   const [content, setContent] = useState([]);
   const db = getFirestore(firebaseApp);
 
+  const { user, userInDb } = useFirebaseAuth();
+  const [viewerIsAuthor, setViewerIsAuthor] = useState(false);
+  const [editingName, setEditingName] = useState(false);
+  const [editingUserName, setEditingUserName] = useState(false);
+  const [editingDescription, setEditingDescription] = useState(false);
+  const [editingUsername, setEditingUsername] = useState(false);
```
Comment (ynadeem) [inline pages/[username].js:31]: Duplicate line of code

**Review by ynadeem — CHANGES_REQUESTED** [review]
Please see my comment on line 31 of pages/[username].js

**Comment by sbeleidy** [comment]
Good catch! Changed

**Review by ynadeem — APPROVED** [review]
Looks good

**Pull request merged**